### DISCUSSION TITLE
Tailscale

### DISCUSSION THREAD

**Discussion by GanglesXIII** [Atlas-OS/Atlas#1249] [Help & Support] [answered]
I can't seem to get tailscale to work on my pc even with fresh install. Login hangs from web browser and from app. this doesn't happen on any other windows pc i've tried.  so i was wondering if there is something different at OS level that could affect this connection.

**Comment by Ast3risk-ops** [comment]
> I can't seem to get tailscale to work on my pc even with fresh install. Login hangs from web browser and from app. this doesn't happen on any other windows pc i've tried.  so i was wondering if there is something different at OS level that could affect this connection.  

Check you can access login.tailscale.com, the Tailscale log endpoint, and the Tailscale control plane.

**Reply by GanglesXIII** [reply]
I did check before fresh install with no success, will try again on the fresh install when i get back to it.

**Reply by GanglesXIII** [reply]
Tried again, same results, nothing happens until eventually connection times out. Cant be router config as other machines on the network work. So it has to be something with this machine. Its my only machine running atlas at the moment but if noone else is having issues, it might just be a coincidence. 
Tailscale's troubleshooting has been of little help also or i wouldn't be asking here. I havent even went through all the config options of atlas yet, still default. Maybe a bios setting needs to be changed? Any insight would be apreciated. If all we do is confirm its not an atlas issue, it all helps. Thanks!

**Reply by Ast3risk-ops** [reply]
try enabling VPN support (in the atlas folder) or running the reset network to windows default script (also in the atlas folder).

**Reply by GanglesXIII** [reply, marked as answer]
i could not find the vpn support, but after all kinds of fiddling it seems the problem lies within my ethernet connection. I enabled the wifi connection and it worked.  so perhaps it is something in my router after all.  i am plugged into a switch so maybe some extra configuration is required. Thanks for the help though.
